src/components/Checkbox/checkbox.tsx

```diff
@@ -1,26 +1,22 @@
 "use client";
 
 import React, { useState } from "react";
-import Style from "./Checkbox.module.css";
+import styles from "./Index.module.css";
 
 interface CheckboxProps {
```

Comment:
When extending ComponentProps you want to remove `type`, since for this checkbox component the only possible type is `checkbox` which we define in the component itself. 

In typescript you can remove a property from an interface by using `Omit`, for example:


```
 interface CheckboxProps extends Omit<ComponentProps<"input">, "type"> {}
```